pandas/io/formats/format.py

```diff
@@ -217,16 +223,19 @@ def __init__(
 
         self._chk_truncate()
 
-    def _chk_truncate(self):
+    @property
+    def truncate_v(self) -> bool:
```

Comment:
Have to excuse my lack of knowledge with this module but this isn't entirely consistent with the previous logic right? Any risk in implementing this differently?

Reply:
removed properties